src/models/extensions/defaultExtension.ts

```diff
@@ -0,0 +1,8 @@
+import { FileFormat } from './fileFormat';
+
+export interface IDefaultExtension {
+  icon: string; // name of the icon.
+  format: FileFormat | string; // format of the icon
+  _custom?: boolean; // user customization: flag for the icon generator.
```

Comment:
I currently working on a way to remove `_custom`. Implementation has finished but have to find a way to make tests pass.